### ISSUE TITLE
Technical improvements for CVs module

### ISSUE THREAD

**Issue by eamelink** [lunatech-labs/lunatech-competence-center-online#73]
Currently there are several issues with the CV module:

1. Not really related to the CV module itself, but in the top-level menu we now have to routes starting with `/cvs`. The main router doesn't really support that; basically every entry in the left menu should have a different first path segment. Not ideal, and something that we can probably fix, but for now the easy way is to just prevent that we have duplicate first-path-segments.
2. @erikwj and me discussed the datamodel of CVs offline. The outcome was a split between 'Developer Passport', which is the data that a developer fills out about himself, like education, work experience, skills etc. This is the bulk of the data that will end up in the CV. And 'CVs', which is the tooling to list CVs, create CVs etcetera. The 'CVs' page should normally not be used by developers and I guess should also be hidden from them (but let's postpone the hiding part until we have roles in place in the frontend)
3. The CVs page does imperative routing internally, but it should embed an `<app-route>` component for the routing. 

So, to summarize the solutions:
* Split off the  `/cvs/me` page into a separate "Developer Passport" component and page
* Give that new page the url `/developer-passport` or something like that. 
* Improve the routing mechanism inside `page-cvs` to use an `<app-route>` instead of the current imperative routing. I'm willing to help with this.

**Comment by eamelink** [comment]
First two are done and merged. Third one is discussed with @erikwj.
